examples/rest-api/client/script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
```

Comment:
I understand.  But *we want the script to be able to fail*.

In the Readme.md, we do:

```
kubectl delete AzureIdentityBinding client-principal-binding --namespace pir
```

To show that the token-grabbing fails while the binding is deleted.  We then recreate it...

Reply:
Actually…  I was going over the corrections.  Since I haven’t touched this for a while, I decided to blindly follow my README.md to test it.

It turns out that a behaviour seems to have changed…

Towards the end I was suggesting to look at the client fetching a token and calling the REST API every second.  At the same time, I was deleting the AzureIdentity to show the token fetching was failing right away.

…  that isn’t the case anymore.  The token fetched on http://169.254.169.254/metadata/identity/oauth2/token/ keeps returning no problem.

Is it possible that you guys changed that behaviour so it is now cached?

If so, is that the “final behaviour”, in which case, I would remove the section suggesting to delete to then recreate the AzureIdentity since it doesn't do much now...